packages/client/src/views/SysAdmin/Config/Integrations/Integrations.tsx

```diff
@@ -0,0 +1,104 @@
+import { withOnlineStatus } from '@client/views/OfficeHome/LoadingIndicator'
+import { connect } from 'formik'
+import React, { useState } from 'react'
+import { Content } from '@opencrvs/components/lib/Content'
+import { injectIntl, useIntl } from 'react-intl'
+import { messages } from '@client/i18n/messages/views/formConfig'
+import {
+  LinkButton,
+  SecondaryButton
+} from '@client/../../components/lib/buttons'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/ListViewSimplified'
+import { Frame } from '@opencrvs/components/lib/Frame'
+import { Navigation } from '@client/components/interface/Navigation'
+import { Header } from '@client/components/Header/Header'
+import {
+  AvatarSmall,
+  Plus,
+  VerticalThreeDots
+} from '@client/../../components/lib/icons'
+import { Pill, ToggleMenu } from '@client/../../components/lib'
+
+interface IIntegration {
+  name: string
+  status: string
+  systemId: string
+}
+
+export const statuses = {
+  PENDING: 'pending',
+  ACTIVE: 'active',
+  DISABLED: 'disabled',
+  DEACTIVATED: 'deactivated'
+}
+
+export function IntegrationList() {
+  const intl = useIntl()
+
+  return (
+    <Frame
+      header={<Header />}
+      navigation={<Navigation loadWorkqueueStatuses={false} />}
```

Comment:
lets pull `ocrvs-2277-integration-configuration` branch, and add

```
        skipToContentText={intl.formatMessage(
          constantsMessages.skipToMainContent
        )}
```
This is for accessibility, so that keyboard users can ["Skip to content"](https://bignerdranch.com/blog/web-accessibility-skip-navigation-links/)
you also need `import { constantsMessages } from '@client/i18n/messages'` import then.

Reply:
updated.